src/redux/homePage/homePage-reducer.test.ts

```
@@ -0,0 +1,62 @@
+import homePageReducer, { initialState } from './homePage-reducer';
+import * as actionTypes from './homePage-actions';
+
+describe('homePage reducer', () => {
+  it('GET_MOVIES_LOADING without error message', () => {
```

Вместо it можешь test использовать, лучше читается)
Просто обычно с it тест выглядит так
it('should throw an error', () => {